app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
Using `self.people[self.name] = self` will raise an error because `self.people` is trying to access an instance variable, but `people` is defined as a class variable. You should use `Person.people[self.name] = self` to correctly reference the class variable.